src/main/java/uk/gov/companieshouse/acsp/service/FilingsService.java

```diff
@@ -101,9 +101,9 @@ private void buildFilingStatus(FilingApi filing) {
   private void buildPresenter(Map<String, Object>data, AcspDataDto acspDataDto) {
     var presenter = new Presenter();
 
-    presenter.setFirstName(Optional.ofNullable(acspDataDto.getFirstName())
+    presenter.setFirstName(Optional.ofNullable(acspDataDto.getApplicantDetails().getFirstName())
```

Comment:
you will have to check if applicant details is null or not else its a potential NPE

Reply:
I would say create a method like buildApplicantDetails where you build the details.

Reply:
Done